### PR TITLE
Feature/AUT-1299/add icon add-subsection

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/AUT-1299

Added `add-subsection` icon

Depends on:
- [x] https://github.com/oat-sa/tao-core-ui-fe/pull/399

### CODE REVIEW

**Review by Silvia-loza — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [X] Pull request's target is not master
- [X] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful

File: views/package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
         "@oat-sa/tao-core-libs": "0.4.4",
         "@oat-sa/tao-core-sdk": "1.16.1",
         "@oat-sa/tao-core-shared-libs": "1.0.3",
-        "@oat-sa/tao-core-ui": "1.44.0",
+        "@oat-sa/tao-core-ui": "git://github.com/oat-sa/tao-core-ui-fe.git#feature/AUT-1299/add-subsection",
```
Comment (adriloma) [inline views/package.json:20]: We better wait to merge until npm package is published and we update this dependency

**Review by adriloma — APPROVED** [review]
Approved, but I suggest postponing the merge until npm dependency is updated and package.json does not point to the temporary branch.

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following conventional commits
- [x] Commits messages are meaningful
- [x] Commits are atomic

**Comment by AntonNeviarouski** [comment]
QA verified

**Pull request merged**